tensorflow/contrib/sparsemax/python/ops/sparsemax_loss.py

```diff
@@ -69,7 +69,7 @@ def sparsemax_loss(logits, sparsemax, labels, name=None):
     # The code below also coveres the case where z = inf. Howeverm in this
     # caose the sparsemax will be nan, which means the sum_s will also be nan,
     # therefor this case doesn't need addtional special treatment.
-    q_part_safe = array_ops.where(
+    q_part_safe = array_ops.where_v2(
```

Comment:
The tests for this are failing because they rely on the old broadcasting behavior of tf.where. Please fix.

Reply:
@alextp changes are reverted for this file, kindly check.

Regards
Amit